### ISSUE TITLE
TMDb scraping bug

### ISSUE THREAD

**Issue by levander051** [Komet/MediaElch#653]
When batch scraping writers with TMDb many get un-scraped. When scraping individually there is no problem.

**Comment by bugwelle** [comment]
Yes, that's the current behavior of MediaElch, even though no a best-practice. :/
Will see when I've got time to fix this.
